### ISSUE TITLE
installer: unable to install on legacy bios, only uefi

### ISSUE THREAD

**Issue by messi-23** [linuxmint/mint21.1-beta#25]
When I'm the default erase-and-install method with legacy mode, it get a superfluous EFI partition
![image](https://user-images.githubusercontent.com/120046409/206245740-2a429385-08e3-4cfe-afc3-17d8fb977b2c.png)

**Comment by clefebvre** [comment]
That shouldn't matter. That's the way ubiquity or at least partman-auto was designed upstream.

Are you saying it doesn't install/boot or something?

**Comment by messsi-23** [comment]
@clefebvre yes, Yes, I can't start up or boot, it happend after mint 21 and 21.1, on 20.3 it works fine
